lib/device/lib/logging.js

```diff
@@ -143,13 +143,26 @@ Logging.prototype.mark = function(name, time) {
  */
 Logging.prototype.memory = function(pid, context) {
   var logging = this;
+  var marionette = this.device.marionette;
   pid = pid.toString();
 
-  return this.device.command()
+  var command = this.device.command()
     .env('ANDROID_SERIAL', this.serial)
     .adbShell('b2g-info')
-    .pipe('grep "' + pid + '"')
-    .exec()
+    .pipe('grep "' + pid + '"');
+
+  return marionette
+    .startSession()
+    .then(function(client) {
+      return marionette
+        .triggerGC()
```

Comment:
I think it would be good to have a debug entry here, so in the console it will show that a GC was performed